### DISCUSSION TITLE
Support instrumentation through openllmetry

### DISCUSSION THREAD

**Discussion by baggiponte** [langfuse/langfuse#2043] [Ideas]
### Describe the feature or potential improvement

[openllmetry](https://github.com/traceloop/openllmetry) by Traceloop allows instrumenting your LLM applications and send data to any collector such as splunk, datadog, signoz and so on. It's a standardisation effort that's becoming quite popular and successful and I think langfuse would benefit from supporting that.

This would mean that most applications already using openllmetry/opentelemetry could seamlessly send data to langfuse (or a different product, like the APM posthog). This could tie langfuse closer to the telemetry ecosystem. It would also reduce the duplication effort of instrumenting the same app for two different collectors.

**Comment by marcklingen** [comment]
Thanks again for raising this, let's centralize the discussion on this other thread as it has been more active: #2509
